.editorconfig

```
@@ -0,0 +1,5 @@
+[*.{cs,vb}] 
```

Do we need VB files? Seems odd to me. Feel free to ignore if important.

```suggestion
[*.cs]
```

---

No idea but I don't think it hurts